felix/calc/states_for_test.go

```diff
@@ -1700,11 +1800,29 @@ var vxlanV6WithBlock = empty.withKVUpdates(
 	},
 ).withExpectedEncapsulation(
 	proto.Encapsulation{IpipEnabled: false, VxlanEnabled: true},
-)
+).withRoutes(vxlanV6WithBlockRoutes...)
+
+var vxlanV6WithBlockRoutes = []proto.RouteUpdate{
+	routeUpdateV6IPPoolVXLAN,
+	routeUpdateRemoteHostV6,
+	// Single route for the block.
+	proto.RouteUpdate{
+		Type:        proto.RouteType_REMOTE_WORKLOAD,
+		IpPoolType:  proto.IPPoolType_NONE,
```

Comment:
Shouldn't this IP pool type be VXLAN? And same for all of the test states that build off of this?

Reply:
Same, I think this may point to wrong product code, I'm going to look into this. Thanks for catching it!

Reply:
Not a problem with product code, fixed the test code.